stable/weave-cloud/templates/NOTES.txt

```diff
@@ -1,10 +1,10 @@
-{{- if .Values.ServiceToken -}}
+{{- if .Values.token -}}
```

Comment:
You will need to change the logic here to check for the default value.

Reply:
If I'm checking for the default value the e2e tests will fail 😢

Reply:
interesting... it sounds like we should really get someone who knows more about e2e tests to help us!

Reply:
Using the CI dir solved this, the default values contain an empty token now.